src/web/jcc/client/components/SetHubLocationPanel.tsx

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/sort-comp */
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect, useRef } from 'react'
+import Button from '@mui/material/Button';
+import {JaiaAPI} from '../../common/JaiaAPI'
+import { PortalHubStatus } from './shared/PortalStatus';
+import { CommandForHub, GeographicCoordinate, HubCommandType } from './shared/JAIAProtobuf';
+import { getElementById, getGeographicCoordinate } from './shared/Utilities';
+import { Map } from 'ol';
+import PointerInteraction from 'ol/interaction/Pointer';
+import { Select, MenuItem } from '@mui/material';
+
+interface Props {
+    hubs: {[key: number]: PortalHubStatus}
+    api: JaiaAPI
+    map: Map
+}
+
+
+export default function SetHubLocationPanel(props: Props) {
+    const [ hub_id, set_hub_id ] = useState(Number(Object.keys(props.hubs)[0]) ?? 1)
```

Comment:
I decided to go with snake_case, only because hub_id is a field name from a protobuf message.  Our general principle in JS has been to use the snake_case only for protobuf stuff, so those fields have unambiguous names across JS, C++, and python.

@ferrom-jaia what do you think on this one?

Reply:
Ok, switched to hubId